src/recursion.js

```diff
@@ -6,105 +6,247 @@
 // denoted by n!, is the product of all positive integers less than or equal to n.
 // Example: 5! = 5 x 4 x 3 x 2 x 1 = 120
 // factorial(5); // 120
-var factorial = function(n) {
+let factorial = function(n) {
```

Comment:
I suggest preferring `const` over `let`. Use `let` _only_ when a variable needs to be reassigned.

https://eslint.org/docs/rules/prefer-const